contents/blog/best-open-source-session-replay-tools.md

```diff
@@ -0,0 +1,195 @@
+---
+date: 2022-04-07
+title: Open source (and self-hosted) alternatives to Hotjar, FullStory & LogRocket
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-company-culture-blog.png
+featuredImageType: full
+author: ["hanna-crombie"]
+categories: ["Guides", "Product analytics"]
+---
+
+Most analytics tools have one goal in mind: providing actionable insights on user behavior. It's all about hard numbers; views, clicks, conversions. But sometimes all those numbers can feel a bit sterile. Separated from the individual user narrative, statistics are an incomplete picture. Enter session recording.
```

Comment:
This intro is close but not quite there in terms of grabbing me. How about something like:

'Most analytics tools just give you the hard numbers: views, clicks, conversions. This is a good starting point, but statistics alone give you an incomplete picture - _why_ did the user rageclick or fail to convert? Enter session recording.'

(I don't know if this is better - my point is it could be more snappy)

Reply:
Good shout